{{cookiecutter.project_folder}}/pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[tool.poetry]
+name =  "{{ cookiecutter.namespace + '-' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"
+version = "{{ cookiecutter.version }}"
+description = "{{ cookiecutter.short_description }}"
+authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
+readme = "README.md"
+packages = [{include =  "{{ cookiecutter.namespace + '/' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"}]
+
+[tool.poetry.dependencies]
+python = ">={{ cookiecutter.python_version }}, <3.12"
+{%- if cookiecutter.use_jupyterlab == "y" %}
+jupyterlab = "^3.0.0"
+{%- endif %}
+{%- if "{{ cookiecutter.command_line_interface|lower }}" == "cli" %}
+typer-cli = "^0.0.13"
+{%- endif %}
+{%- if cookiecutter.use_mypy == "y" %}
+pydantic = "^1.10.7"
+{%- endif %}
+
+[tool.poetry.group.dev.dependencies]
+black = {version = "^23.1.0", allow-prereleases = true}
```

Comment:
Not sure we need black if we have Ruff?

Reply:
No we dont. I have removed "black" from the dev dependencies